sentry-android-core/src/test/java/io/sentry/android/core/SentryAndroidTest.kt

```diff
@@ -392,8 +392,8 @@ class SentryAndroidTest {
         }
     }
 
-    @Test
-    @Config(sdk = [30])
+//    @Test
```

Comment:
Do we need to re-enable this?

Reply:
This should be fixed with merging `main` into this branch. Will re-enable in a follow up PR.